lib/tasks/export.rake

```diff
@@ -206,11 +246,19 @@ namespace :export do
       papers.append(paper)
       reviewer = ReviewerReport.where(task_id: paper.tasks.pluck(:id)).map(&:user).uniq.compact.sample
       paper2 = reviewer.reviewer_reports.first.try(:paper) unless reviewer.nil?
-      papers.append(paper2) unless paper2.nil?
-    end
-    papers.each do |paper|
-      export_paper(paper)
+      papers.append(paper2) unless paper2.nil? || papers.include?(paper2)
     end
+
+    paper_queue = Queue.new
+    papers.each { |paper| paper_queue << paper}
```

Comment:
so the dirty way to do this is `papers.inject(Queue.new, :push)`

Reply:
oh yeah, nice...